Sources/Contentful/Date.swift

```diff
@@ -32,13 +32,12 @@ public extension Date {
     }
 
     /// A formatter ready to handle iso8601 dates: normalized string output to an offset of 0 from UTC.
-    public static func iso8601Formatter() -> DateFormatter {
+    public static func iso8601Formatter(timeZone: TimeZone? = nil) -> DateFormatter {
         let formatter = DateFormatter()
         formatter.calendar = Calendar(identifier: .iso8601)
         // The locale and timezone properties must be exactly as follows to have a true, time-zone agnostic (i.e. offset of 00:00 from UTC) ISO stamp.
         formatter.locale = Foundation.Locale(identifier: "en_US_POSIX")
-        formatter.timeZone = TimeZone(secondsFromGMT: 0)
-        // Normalize to 0 UTC offset.
+        formatter.timeZone = timeZone ?? TimeZone(secondsFromGMT: 0)
```

Comment:
Can you set the default value instead of to `nil` to `TimeZone(secondsFromGMT: 0)` directly?

Reply:
Yes, I could have done that @dlitvakb but i just chose to do it this way to show the expose a public interface for this function that takes an optional timeZone.